picasso/src/main/java/com/squareup/picasso3/Picasso.java

```diff
@@ -64,7 +64,7 @@
  * Use {@see PicassoProvider#get()} for a global singleton instance
  * or construct your own instance with {@link Builder}.
  */
-public class Picasso implements LifecycleObserver {
+public class Picasso implements LifecycleObserver, EventListener {
```

Comment:
This is a bit weird because it means you could pass a Picasso instance as an implementation to another as an event listener. I think you just want package-protected copies of the methods without implementing the interface.

Reply:
Makes sense.  Updated.